tips/views.py

```diff
@@ -20,6 +21,16 @@ def add_tip(request):
     return render(request, ADD_TIP_PAGE_ROUTE, {'form': form})
 
 
+def categories_filter(request, category):
```

Comment:
I think the name "tip_filter_by_category" would fit better here 👍 
The tips are filtered here, not the categories. They are filtered by the categories